site/content/en/docs/manual/basics/task-details.md

```diff
@@ -43,6 +43,7 @@ and the details of the task (specified when the task was created) and the jobs s
   - Duration — is the amount of time the job is being worked.
   - Assignee is the user who is working on the job.
     You can start typing an assignee’s name and/or choose the right person out of the dropdown list.
+    If you have not chosen any [organization] (/docs/manual/advanced/organization) or you have not chosen the assignee, the job is assigned to you.
```

Comment:
Could you please explain what you mean here?

Reply:
Well, my user case was: 

1. I sign up.
2. I do not choose or create any organization.
3. I create a new task.
4. I go to ‘Jobs’ assigned to me. The task is there.